ci-operator/config/red-hat-data-services/notebooks/red-hat-data-services-notebooks-release-2024b.yaml

```diff
@@ -0,0 +1,317 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.17-openshift-4.10
```

Comment:
I suppose it's not a big deal, but maybe we should consider to update it to something newer?

Reply:
Good point, I updated to `rhel-9-release-golang-1.21-openshift-4.16`